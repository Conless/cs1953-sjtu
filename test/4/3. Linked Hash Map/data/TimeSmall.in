1
13 0 10 
0 3 7BQEzo8knH24pxK9hOqm 
1 3 7BQEzo8knH24pxK9hOqm 
0 6 zr9TUZ3LAPytFkgsJdYx 
2 6 
1 6 zr9TUZ3LAPytFkgsJdYx 
0 10 5I3ambDGFCfc8unPhRz6 
1 10 5I3ambDGFCfc8unPhRz6 
0 7 rlQpdvVZyYWH9gz5UasL 
1 7 rlQpdvVZyYWH9gz5UasL 
0 6 xnXaD6tRGdQEM8oSghKJ 
