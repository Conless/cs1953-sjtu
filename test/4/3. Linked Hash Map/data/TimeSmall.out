zr9TUZ3LAPytFkgsJdYx 
6 xnXaD6tRGdQEM8oSghKJ
